man/el.man

```diff
@@ -0,0 +1,60 @@
+.TH VRMS-RPM 1
+.SH NAME
+vrms-rpm - αναφορά εγκατεστημένου μη-ελεύθερου λογισμικού
+.SH ΣΥΝΟΨΗ 
+\ fBvrms-στροφές \ fR \ fI [ΕΠΙΛΟΓΗ] \ fR ...
```

Comment:
`\fB`, `\fI` and `\fR` are formatting marks - they switch between bold, italic and roman ("normal") text. The spaces after the slashes must be removed.